activerecord-virtual_attributes.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
 
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "activerecord", "~> 6.1.0"
+  spec.add_runtime_dependency "activerecord", ">=6.1.7.6"
```

Comment:
agreed, this needs to be:

```suggestion
  spec.add_runtime_dependency "activerecord", "~>7.0.8"
```

Reply:
and then also drop the 6.1 test suites.

Reply:
will do this once 6.1/7.0 get feature parity.